.warden/commands/bootstrap.cmd

```diff
@@ -239,7 +239,7 @@ elif [[ ${CLEAN_INSTALL} ]]; then
 
   INSTALL_FLAGS="${INSTALL_FLAGS} \
     --cleanup-database \
-    --backend-frontname=backend \
+    --backend-frontname=admin \
```

Comment:
This should probably pair with an issue in the docs repo to update the references to `backend`.

Reply:
Also requires update to line 29 https://github.com/wardenenv/warden-env-magento2/blob/51e4c074ac898af7d2b484142185dd7ad35adfc7/.warden/commands/bootstrap.cmd#L29

Reply:
Should update this repo's readme.md table for AdminURLs all reference backend as well.